src/use-async-task.js

```diff
@@ -49,16 +49,17 @@ export const useAsyncTask = (func, deps) => {
   useEffect(() => {
     let dispatchSafe = action => dispatch(action);
     let abortController = null;
-    const start = async () => {
+    const start = async (...args) => {
       if (abortController) return;
       abortController = new AbortController();
       dispatchSafe({ type: 'start' });
       try {
-        const result = await func(abortController);
+        const result = await func(abortController, ...args);
         dispatchSafe({ type: 'result', result });
       } catch (e) {
         dispatchSafe({ type: 'error', error: e });
       }
+      abortController = null; // allow to run task multile times
```

Comment:
@dai-shi at least consider this line with resetting abortController variable to allow restarting async task

Reply:
@sergeyt With my current assumption, this is intentional to avoid restarting the async task.
Again, I might misunderstand something, and I would appreciate for your use case/example. It doesn't have to be with formik.

The way I think is like:
```javascript
let asynTask = new Promise(resolve => { resolve('foo'); });
asyncTask.then(result => { console.log(result); });
// and then you can't restart this asyncTask
asyncTask = new Promise(...); // but rather create a new one
```

My understanding could be too limited and that's why I'd like example code.

Reply:
@dai-shi your intentional limitation is reasonable too and I'd like to end this thread and my use cases are theoretical :)

Reply:
Alright!